controllers/main.js

```diff
@@ -31,12 +44,34 @@ module.exports.renderPlaySettings = (req, res) => {
   });
 };
 
+module.exports.renderContinue = async (req, res) => {
+  if (!req.user._id) {
+    res.redirect("/play-settings");
+  }
+  const user = req.user._id;
+  const lastSessionData = await fetchPlaySessionData(user);
+  const image = await getPixelatedImage(
+    lastSessionData.sessionData[lastSessionData.sessionData.length - 1].imgLink
+  );
+  const sessionData = {
+    difficulty: remapDifficulty[lastSessionData.difficulty],
+    length: lastSessionData.length,
+    progress: lastSessionData.sessionData.length,
+    id: lastSessionData.sessionId,
+    image,
+  };
+
+  res.render("main/continue.ejs", { extraStyles: cardStyle, sessionData });
+};
+
 module.exports.renderPlay = async (req, res) => {
   const { id, pageNum } = req.params;
   const playSessionData = await PlaySession.findOne({ sessionId: id });
   let pageGameData;
+  const userGuessText =
```

Comment:
What does `userGuessText` mean? As far as I know the user cannot type in a guess value (at least not without devtools lel) so this name doesn't make a lot of sense

Reply:
We already have a `userGuess` variable in our session which is a `Boolean`, indicating whether the user has guessed correctly or not.
`userGuessText` saves the name of the game that the user has selected as his answer. This is later used to autofill the form when revisiting previous guess pages.

Only now i do realize that instead i probably could have simply saves the index'ed input id and worked with that.